core/sources/utf8_utils.move

```diff
@@ -21,6 +51,10 @@ module aptos_names::utf8_utils {
         let len = vector::length(&u64_characters);
         let allowed = true;
 
+        if (ends_with(&u64_characters, &MOVE_DOMAIN_SUFFIX)) {
```

Comment:
what if people try to register a name like `charles.move.apt`?

Reply:
Perhaps the policy should be:
1. Dots are banned in names.
2. For convenience, if there is a single dot in the name and the suffix is in the valid set, remove the dot and suffix.
3. If there is more than 1 dot, abort.

Reply:
if i understand the question correctly, `charles.move.apt` would abort because the loop below would see a `.` in the name and return false